### DISCUSSION TITLE
What exactly is the "limited feature set" of private repos in free organisation accounts?

### DISCUSSION THREAD

**Discussion by alimony** [community/community#22072] [New to GitHub]
<p>I’ve noticed protected branches and wikis are two features missing from private repos in the free organisation account type, but is it documented anywhere exactly what the feature set is, i.e. the difference in features between private and public repos in organisation accounts.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/alimony">@alimony</a>! :wave: Welcome to the Community!</p>
<p>In the <a href="https://docs.github.com/en/github/getting-started-with-github/faq-about-changes-to-githubs-plans#if-i-downgrade-from-github-team-or-a-legacy-plan-to-github-free-what-features-will-i-lose">help article</a> about downgrading your account from a paid plan to a free plan it lists the features you’d lose:</p>
<ul>
<li>Required reviewers in private repositories</li>
<li>Protected branches in private repositories</li>
<li>Code owners in private repositories</li>
<li>GitHub Pages in private repositories</li>
<li>Wikis in private repositories</li>
<li>Multiple assignees for issues and pull requests in private repositories</li>
<li>Multiple pull request reviewers in private repositories</li>
<li>Draft pull requests in private repositories</li>
<li>Autolinked references in private repositories</li>
<li>Standard support</li>
</ul>
